dts/riscv/renode_riscv32_virt.dtsi

```diff
@@ -50,6 +50,13 @@
 			reg = <0x2000000 0x10000>;
 		};
 
+		mtimer: timer@200bff8 {
+			compatible = "riscv,mtime-timer";
+			interrupts-extended = <&hlic 7>;
+			reg = <0x200BFF8 0x8 0x2004000 0x8>;
```

Comment:
If I'm not mistaken, you need to use lower case on all hex entries

```diff
-0x200BFF8
+0x200bff8
```

Reply:
Done.